DEPRECATIONS.md

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Intentionally left blank lines?

Reply:
nope - we can clip them in the first PR :wink: